src/app/charts/backgroundprocessor/templates/deployment.yaml

```diff
@@ -35,13 +34,9 @@ spec:
             - '-1'
           initialDelaySeconds: 5
           periodSeconds: 10
-        resources:
-          limits:
-            cpu: "0.5"
-            memory: "1Gi"
-          requests:
-            cpu: "0.5"
-            memory: "512Mi"
+        {{- if .Values.resources }}
```

Comment:
should this even have an if? When this is not specified, deployment should fail

Reply:
The currently available policies do not really solve that. You can specify limits, but you cannot make `resources` mandatory. For now I'd like to keep it as it is. Our helm charts are considered to follow "best practices" and I'd not rely on any existing or not existing expressions inside the helm chart to fail a deployment.

Especially as no one would prevent a user from removing resources from the `values.yaml` as well as from the `templates/deployment.yaml`.